server/nodetypes/nodetypes.js

```diff
@@ -32,7 +32,7 @@ require ('./lib/init.js')();
 require ('./lib/filesystem.js')();
 
 // Default nodeTypes json files
-var nodeTypesPath = "/usr/share/soletta/flow/descriptions";
+var nodeTypesPath = "/home/phoenix007/gsoc/soletta/build/soletta_sysrootsoletta/share/soletta/flow/descriptions";
```

Comment:
In the patch, let's use the default path from Soletta

also you can use the setPath function from this lib instead of replacing this default path